ee/clickhouse/queries/trends/util.py

```diff
@@ -98,16 +99,22 @@ def get_active_user_params(filter: Filter, entity: Entity, team_id: int) -> Dict
     return params
 
 
-def populate_entity_params(entity: Entity, table_name: str = "") -> Tuple[Dict, Dict]:
+def populate_entity_params(
```

Comment:
The placement of this function is kind of suspect - it's in trends but used winder than that. Worth moving to `ee/clickhouse/models/entity.py`?